docs/ENVIRONMENT.md

```diff
@@ -632,6 +632,14 @@ SYSTEMD_HOME_DEBUG_SUFFIX=foo \
   file (containing firmware measurement data) to read. This allows overriding
   the default of `/sys/kernel/security/tpm0/binary_bios_measurements`.
 
+`systemd-sleep`:
+
+* `$SYSTEMD_SLEEP_SKIP_FREEZE_USER_SESSIONS` - Takes a boolean. When true,
```

Comment:
please invert this, i.e. drop the `_SKIP` from the name, and default to true then,

Reply:
Can do, but I originally inverted this at the request of an earlier review from @YHNdnzj: https://github.com/systemd/systemd/pull/30612#discussion_r1435869747

Reply:
hmm, for most of the knobs i added i really avoided the negated options, and didn't want to convey defaults in the knob names, already so that we have the freedom to change the defaults whenever we like without this having to affect the naming of the env vars.